src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/PropertyPages/ProjectDebugPropertyPage.xaml

```diff
@@ -26,7 +26,15 @@
 
   <StringProperty Name="CommandLineArguments"
                   DisplayName="Command line arguments"
-                  Description="Command line arguments to pass to the executable." />
+                  Description="Command line arguments to pass to the executable.">
```

Comment:
Maybe explain that multi-line is acceptable here:

```suggestion
                  Description="Command line arguments to pass to the executable. You may break arguments into multiple lines.">
```

Reply:
Changed.